sys-fs/zfs-auto-snapshot/zfs-auto-snapshot-1.2.4.ebuild

```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/zfsonlinux/${PN}.git"
+else
+	SRC_URI="https://github.com/zfsonlinux/${PN}/archive/upstream/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64"
+	S="${WORKDIR}/${PN}-upstream-${PV}"
+fi
+
+DESCRIPTION="ZFS Automatic Snapshot Service for Linux"
+HOMEPAGE="https://github.com/zfsonlinux/zfs-auto-snapshot"
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="+default-exclude"
+
+RDEPEND="sys-fs/zfs
+		virtual/cron"
+
+src_install() {
+	if use default-exclude; then
+		for cronfile in etc/${PN}.cron.{daily,hourly,monthly,weekly}; do
+			sed -i "s/\(^exec ${PN}\)/\1 --default-exclude/" "$cronfile" || die
+		done
+		sed -i "s/\(; ${PN}\)/\1 --default-exclude/" etc/"${PN}".cron.frequent
+	fi
+	emake PREFIX="${EPREFIX}/usr" DESTDIR="${D}" install
+}
+
+pkg_postinst() {
+	if ! use default-exclude; then
```

Comment:
Optional suggestion: use `readme.gentoo-r1` eclass to additional store that in `README.gentoo` for future read.